<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebitCalcView</class>
 <widget class="QWidget" name="DebitCalcView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLineEdit" name="period_le">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="capitalization_cb">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>добавлять ко вкладу</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>выплачивать</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Доход за вычетом налогов:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Сумма вклада:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>мес.</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Сумма довложений:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>          </string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QComboBox" name="rep_cb">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>нет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>раз в месяц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>раз в полгода</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>раз в год</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="sum_le">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="calc_pb">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="percent_le">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="curr_qb"/>
   </item>
   <item row="7" column="3">
    <widget class="QLineEdit" name="rep_sum_le">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDateEdit" name="dateEdit">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="3" rowspan="7" colspan="4">
    <layout class="QGridLayout" name="donut_layout1">
     <item row="1" column="1">
      <widget class="QLabel" name="rep_sum_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="tax_paid_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="sum_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="profit_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string> </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="percent_acc_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string> </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="7">
      <layout class="QGridLayout" name="donut_layout" rowminimumheight="0"/>
     </item>
    </layout>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Начисленные проценты:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>% годовых</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Удержано налогов:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="percent_perio_cb">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>в конце срока</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>раз в месяц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>раз в полгода</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>раз в год</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="26" column="1" colspan="6">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="25" column="1">
    <widget class="QPushButton" name="pays_graph_btn">
     <property name="text">
      <string>График выплат</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_18">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 5px;  min-height: 10px;  min-width: 10px;background-color: orange;</string>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Сумма пополнения:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Периодичность пополнения вклада:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="percent_label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Начислено процентов:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Процентная ставка:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Дата начала срока:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Срок размещения:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Сумма вклада:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_19">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">  border-radius: 5px;  min-height: 10px;  min-width: 10px;background-color: green;</string>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_20">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">  border-radius: 5px;  min-height: 10px;  min-width: 10px;background-color: cyan;</string>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_21">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">  border-radius: 5px;  min-height: 10px;  min-width: 10px;background-color: red;</string>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
